programs/client/Client.cpp

```diff
@@ -319,6 +319,10 @@ void Client::initialize(Poco::Util::Application & self)
     /// Set the path for google proto files
     if (config().has("google_protos_path"))
         client_context->setGoogleProtosPath(fs::weakly_canonical(config().getString("google_protos_path")));
+
+    /// Use <warnings/> unless --no-warnings is specified
```

Comment:
Can we add a basic test for this?

Reply:
I'll look to see if the client has any tests that this could be added to

It seemed a trivial enough change, literally two lines, but maybe it's not so much more for a simple test